### DISCUSSION TITLE
Busco equipo con ganas de desarrollar un lindo proyecto

### DISCUSSION THREAD

**Discussion by julianFanjul** [Launch-X-Latam/MisionFrontEnd#518] [Team Posts]
Hola a todos, soy Julian, de Argentina, soy diseñador multimedial con conocimiento en herramientas de diseño y HTML, CSS, REACT, NODE,JS, MONGODB , THREE.JS, entre otros..

Busco compañeros que quieran trabajar codo a codo para poder presentar proyecto interesante.
Me pueden escribir por teams o comenten por aca su email

Dejo mi email : lx06200@innovaccion.mx

**Comment by snowingt** [comment]
Hola si aun no tienes me gustaria participar este es mi correo escribeme snowingtorres@gmail.com

**Comment by 4Ndrs91** [comment]
Hola quisiera unirme si hacen un equipo este es mi correo lx12963@innovaccion.mx

**Comment by vanessaurbina-tuc** [comment]
Hola! Aqui Vanessa tamb de Argentina, mas claro de Tucuman. Soy dis ux/ui y somos dos buscando equipo. Ambos contamos con conocimientos en Frontend. Encantados de formar equipo 

Si gustas, dejo nuestros correos :

 lx05855@innovaccion.mx (Matias)
 lx05862@innovaccion.mx (Vanessa)

**Comment by Gallardo1225** [comment]
Me gustaría formar parte del equipo.
Daré lo mejor de mi

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by PepoGE** [comment]
Hola! Me llamo José Luis y estaría encantado de formar parte del equipo, ando cursando una Ingeniería en TI,  tengo conocimientos en FrontEnd, programación en Java y experiencia con UI, siempre dispuesto a aprender y ayudar con lo que pueda aportar :D 
Mi correo es: lx03228@innovaccion.mx

**Comment by mysticBel** [comment]
Hola , te habia enviado un correo.. espero te haya llegado !

**Comment by NArAr12** [comment]
Hola, me llamo Natalia quisiera formar parte de tu equipo.
Mi correo lx10738@innovaccion.mx
